test/common/http/conn_manager_impl_test.cc

```diff
@@ -74,7 +74,8 @@ class HttpConnectionManagerImplTest : public Test, public ConnectionManagerConfi
                                         POOL_TIMER(fake_stats_))},
                "",
                fake_stats_},
-        tracing_stats_{CONN_MAN_TRACING_STATS(POOL_COUNTER(fake_stats_))} {
+        tracing_stats_{CONN_MAN_TRACING_STATS(POOL_COUNTER(fake_stats_))},
+        listener_stats_{CONN_MAN_LISTENER_STATS(POOL_COUNTER_PREFIX(fake_stats_, "http.fake."))} {
```

Comment:
nit: for consistency I would make this use the same prefix as above which is "".

Reply:
Won't fix. If I were to do that, the Http Conn Man stats and the per listener stats would be under the same scope and the values for rq_2xx will double. I think this would make the tests confusing.

I can change the prefix in the constructor to be `listener._.http.fake.`

Reply:
What you should do is actually make a new scope with some prefix in it, instead of passing the same raw stats object. (Or pass a different isolated store).

Reply:
Passed a different isolated store.